### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
more details about the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # ssis.bit
 AIO solution for SSIS students
+
+## Hardware
+
+CircuitPython supports [more than 200 different boards](https://circuitpython.org/downloads). Locally available is the [TTGO T8 ESP32-S2 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) at several online stores. The board includes the powerful 240MHz CPU, 4MB Flash and 8MB PSRAM, a 1.14" ST7789 display, a microSD card slot for virtually unlimited local data storage, battery connector 1.25mm JST with charge controller and regular USB-C connector. And buildin WiFi, of course.
```
Comment (kreier) [inline README.md:6]: Good description

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+
+The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
```
Comment (kreier) [inline README.md:8]: Please include a picture

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+
+## Software
+
+The ssis:bit runs [CircuitPython](https://circuitpython.org/) and can be easily programmed with the [Mu Editor](https://codewith.mu/en/). There is a lot to learn on [Adafruit with CircuitPython]() and with their great libraries there is less frustration getting things started.
```
Comment (kreier) [inline README.md:12]: Does it work on M1 Macs as well?

Comment (ssis-robotics) [reply README.md:12]: Will be tested next week

File: README.md (hunk 1 above)
Comment (ssis-robotics) [reply README.md:6]: Actual prototype will be build in January 2022

File: README.md (hunk 2 above)
Comment (ssis-robotics) [reply README.md:8]: The webpage has it.

Comment (ssisbit) [reply README.md:8]: Without our daughter board it looks like this:
T8

Comment (ssisbit) [reply README.md:8]: Running CircuitPython it looks like this:
lilygo_ttgo_t8_s2_st7789

Comment (kreier) [reply README.md:8]: can you included it in the about page?

**Pull request merged**